### DISCUSSION TITLE
GnuDb Problem?

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#687] [Bug Reports]
I tried playing an album with add and replace which has had no previous problems. The Playlist is blank, and nothing shows in the album folder. The album plays, but is showing track titles which are incorrect. This information appears to be coming from GnuDb, and is overriding the correct metadata from the file.

What has changed, and how do I stop it?

**Comment by R101G** [comment]
Re-ripping the CD and replacing the album folder with the new rip has restored things to normal, but I would still like to know what could have caused this.

**Reply by rern** [reply]
`http://gnudb.gnudb.org` is for audio CD data. The data should be exposed while playing the CD only.

**Comment by R101G** [comment]
Thanks rern. 
Just to be clear, the online info takes precedence over the metadata in the stored file?
